out/js/credit_card_form.js

```diff
@@ -6,22 +6,28 @@
  * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
  */
 /* global WirecardPaymentPage */
-var ModuleCreditCardForm = (function ($) {
-
+var ModuleCreditCardForm = (function($) {
```

Comment:
```suggestion
var ModuleCreditCardForm = (function ($) {
```
Please don't remove spaces from anonymous functions (same applies for the lines below).

Reply:
Codacy showed an error when the space was there.